satellite/payments/stripepayments/stripe.go

```diff
@@ -329,7 +388,33 @@ func (s *service) GetInvoice(ctx context.Context, id []byte) (_ *payments.Invoic
 		CustomFields: customFields,
 		CreatedAt:    time.Unix(inv.Created, 0),
 	}, nil
+}
+
+// GetProjectInvoices returns all invoices for a particular project
+func (s *service) GetProjectInvoices(ctx context.Context, projectID uuid.UUID) (_ []payments.Invoice, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	stamps, err := s.db.ProjectInvoiceStamps().GetAll(ctx, projectID)
+	if err != nil {
+		return nil, stripeErr.Wrap(err)
+	}
+
+	var invoices []payments.Invoice
+	for _, stamp := range stamps {
+		inv, err := s.GetInvoice(ctx, stamp.InvoiceID)
+		if err != nil {
+			return nil, err
```

Comment:
most probably you should wrap error

Reply:
It’s wrapped already as it is public method

Reply:
then few lines earlier you have case were you are wrapping error from public method `GetAll`

Reply:
Yeah but is `DB` public method, so I shoud wrap it as service `err`, but `s.GetInvoice` is a public method of the same service, so `err` is wrapped accordingly

Reply:
:+1: